src/renderers/shaders/ShaderChunk/morphnormal_vertex.glsl.js

```diff
@@ -5,10 +5,23 @@ export default /* glsl */`
 	// When morphTargetsRelative is false, this is set to 1 - sum(influences); this results in normal = sum((target - base) * influence)
 	// When morphTargetsRelative is true, this is set to 1; as a result, all morph targets are simply added to the base after weighting
 	objectNormal *= morphTargetBaseInfluence;
-	objectNormal += morphNormal0 * morphTargetInfluences[ 0 ];
-	objectNormal += morphNormal1 * morphTargetInfluences[ 1 ];
-	objectNormal += morphNormal2 * morphTargetInfluences[ 2 ];
-	objectNormal += morphNormal3 * morphTargetInfluences[ 3 ];
+
+	#ifdef MORPHTARGETS_TEXTURE
+
+		for ( int i = 0; i < MORPHTARGETS_COUNT; i ++ ) {
```

Comment:
I think in GLSL ES 3 we can use a unform for `morphTargetsCount` instead of a define?

Reply:
Unfortunately not. I need the define for the following statement:
```
uniform float morphTargetInfluences[ MORPHTARGETS_COUNT ];
```
Using a uniform does not work here because of:

> ERROR: 0:188: '' : constant expression required
ERROR: 0:188: '' : array size must be a constant integer expression

Reply:
Ah I see!